Source/CarthageKit/NetworkClient.swift

```diff
@@ -0,0 +1,54 @@
+import ReactiveCocoa
+
+
+public func createURLRequest(URL: NSURL, _ headers: [String: String] = [:]) -> NSURLRequest {
+	let urlRequest = NSMutableURLRequest(URL: URL)
+	urlRequest.allHTTPHeaderFields = headers
```

Comment:
Is this equivalent to what we had before? I'm not sure whether there were any default header fields that we were using.